### PR TITLE
Ft grafana staging access

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
**_WHAT DOES THIS PR DO?_**
- Adds a staging self hosted Grafana instance(we need these alerts too just as with prod)

**_WHAT ISSUES ARE RELATED TO THIS PR?_**
 - Adds the missing staging status notifs

**_HOW DO I TEST OUT THIS PR?_**
- Go to https://staging-platform.airqo.net/grafana/
- You should be able to login with your GitHub account
- You will then be redirected to the Grafana dashboard on staging
image

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 
- https://staging-platform.airqo.net/grafana/

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**
- n/a

### CODE REVIEW

File: k8s/grafana/grafana.ini [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+enabled = true
+allow_sign_up = false
+client_id = << client_id >>
+client_secret = << client_secret >>
+scopes = user:email,read:org
+auth_url = https://github.com/login/oauth/authorize
+token_url = https://github.com/login/oauth/access_token
+api_url = https://api.github.com/user
+allowed_organizations = airqo-platform
+role_attribute_path = contains(groups[*], '@airqo-platform/devops') && 'Editor' || 'Viewer'
```
Comment (123MwanjeMike) [inline k8s/grafana/grafana.ini:22]: The Grafana configuration in use

File: k8s/nginx/staging/virtual-server.yaml [hunk 2]
```diff
@@ -14,6 +14,23 @@ spec:
...
+  host: staging-platform.airqo.net
+  upstreams:
+    - name: grafana
+      service: grafana
+      port: 3000
+  subroutes:
+    - path: /grafana
+      action:
+        pass: grafana
+
```
Comment (123MwanjeMike) [inline k8s/nginx/staging/virtual-server.yaml:33]: [staging grafana virtual server route](https://github.com/airqo-platform/AirQo-api/pull/1276/commits/8aab0cfdf06690b2ddd63af58c09f8d8a57468c3)

File: k8s/nginx/staging/global-config.yaml [hunk 3]
```diff
@@ -30,7 +30,7 @@ data:
       if ($request_uri ~ '(/api/v1/users/forgotPassword)|(/api/v2/users/forgotPassword)') { return 200; }
       if ($request_uri ~ '(/api/v1/users/candidates/register)|(/api/v2/users/candidates/register)') { return 200; }
       if ($request_uri ~ '/airflow/api/v') { return 200; }    
-      if ($request_uri ~ '/argocd') { return 200; }    
+      if ($request_uri ~ '/grafana/api') { return 200; }
```
Comment (123MwanjeMike) [inline k8s/nginx/staging/global-config.yaml:33]: [exclude grafana/api path from gateway auth](https://github.com/airqo-platform/AirQo-api/pull/1276/commits/ffec6e7382e8561e944688cc5a4c51fe7cf5bf4f)

**Review by 123MwanjeMike — COMMENTED** [review]
Guiding comments

File: k8s/grafana/datasources.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    isDefault: true
+  - access: proxy
+    name: prometheus-cloud
+    orgId: 1
+    type: prometheus
+    url: "https://prometheus-prod-10-prod-us-central-0.grafana.net/api/prom"
+    basicAuth: true
+    basicAuthUser: << user >>
+    secureJsonData:
+      basicAuthPassword: << password >>
```
Comment (123MwanjeMike) [inline k8s/grafana/datasources.yaml:19]: [prometheus data sources](https://github.com/airqo-platform/AirQo-api/pull/1276/commits/64d1659b00fe44ffb29159226fecca953cb493bb)

**Review by Baalmart — APPROVED** [review]
LGTM, thanks @123MwanjeMike

**Review by NoahNsimbe — APPROVED** [review]
LGTM

**Pull request merged**